### PR TITLE
media-gfx/freecad: directory correction

### PR DESCRIPTION
**Pull request by bdouxx**
Gentoo-Bug: https://bugs.gentoo.org/show_bug.cgi?id=609476

Package-Manager: portage-2.3.3

### CODE REVIEW

File: media-gfx/freecad/freecad-9999.ebuild [hunk 1]
```diff
@@ -65,7 +65,7 @@ RESTRICT="mirror"
 #		salome-smesh - science overlay
 #		zipio++ - not in portage yet
 
-S="${WORKDIR}/FreeCAD-${PV}"
+S="${WORKDIR}/freecad-${PV}"
```
Comment (SoapGentoo) [inline media-gfx/freecad/freecad-9999.ebuild]: can you just make this
`S=${WORKDIR}/${P,,}`
which is more succinct and a bash 4 feature

**Comment by simonvanderveldt** [comment]
This has already been fixed in https://github.com/bdouxx/gentoo/commit/52f42257de118f8123a1778fc2d3071e25cf26e0 and https://github.com/bdouxx/gentoo/commit/a2c72fec5e3371c69a8d597994a2050e1f7cce16

**Comment by bdouxx** [comment]
Hi

Is there a problem with my pull request?

**Comment by simonvanderveldt** [comment]
> Hi
> 
> Is there a problem with my pull request?

No, this is the correct fix. Could/should be merged. Sorry for the noise, I mistakenly looked at the wrong commits/repo.

Note there are more issues with the current ebuild so it'll still fail (at least it does for me), but at least this part would work again :)

**Pull request closed without merging**